Documentation/ceph-cluster-crd.md

```diff
@@ -1427,7 +1427,7 @@ capabilities = ["read"]
 You can write the policy like so and then create a token:
 
 ```console
```

Comment:
@leseb : I have two doubts
1.) do we really need 
```hcl
path "sys/mounts" {
capabilities = ["read"]
}
```
as part of this policy?

2.) Do we need to mention how encryption keys are created in the vault here?

Reply:
I understood part, rooks internally create an encryption key for the osd encryption

Reply:
> @leseb : I have two doubts
> 1.) do we really need
> 
> ```terraform
> path "sys/mounts" {
> capabilities = ["read"]
> }
> ```
> 
> as part of this policy?

Yes we do.

> 
> 2.) Do we need to mention how encryption keys are created in the vault here?

Reply:
> I guess I understood 2nd question, rooks internally create an encryption key for the osd.
> So creating encryption key via `vault kv put ` is not required to mention here

Correct.